### PR TITLE
Ap/cosmos lab

### PR DESCRIPTION
**Pull request by pvainkteshkumar**
Updated the code to use Cosmos Mongo vCore DB instead for Azure Cognitive Search.

@ajai-d , @thegovind

### CODE REVIEW

File: deploy/infrastructure/cli/deploy.ps1 [hunk 1]
```diff
@@ -4,7 +4,10 @@ param (
     [string]$resourceGroupCount = 1,
     [Parameter(Mandatory = $true)]
     [string]$subscriptionId,
-    [string]$deploymentType = "aca"
+    [string]$deploymentType = "aca",
+    [string]$skipAzureCosmosVectorDB = "false",
+    [string]$adminUsername = "admin",
+    [securestring]$adminPassword = "admin1234"
```
Comment (thegovind) [inline deploy/infrastructure/cli/deploy.ps1:10]: This will definitely flag GH secret scanner. I'm surprised it already did not.

**Review by thegovind — CHANGES_REQUESTED** [review]
Thanks for updating to use SK's vCore connector. Could you fix the hard coded secret in the deployment script?

File: deploy/infrastructure/cli/deploy.ps1 (hunk 1 above)
Comment (pvainkteshkumar) [reply deploy/infrastructure/cli/deploy.ps1]: Updated it.

**Review by thegovind — APPROVED** [review]
thank you, @pvainkteshkumar 🎉

**Pull request merged**